dcpy/models/lifecycle/builds.py

```diff
@@ -101,6 +102,22 @@ class LoadResult(BaseModel, extra="forbid"):
     datasets: dict[str, ImportedDataset]
 
 
+class EventType(StrEnum):
+    BUILD = "build"
+    PROMOTE_TO_DRAFT = "promote_to_draft"
+    PUBLISH = "publish"
+
+
+class EventLog(BaseModel, extra="forbid"):
```

Comment:
I think we want this class to mirror the structure of the postgres table exactly, for both insertion and querying, and maybe even creating the table itself. So I think we should probably either have the `run_details` be a json field in the db, or add columns to the model for `timestamp`, `runner_type`, and `runner`.